rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/RMDBManager.java

```diff
@@ -814,4 +814,14 @@ public RMDBManagerBuffer getBuffer() {
         return rmdbManagerBuffer;
     }
 
+    public List<NodeHistory> getNodesHistory(long windowStart, long windowEnd) {
+        return executeReadTransaction(session -> {
+            Query query = session.createQuery("FROM NodeHistory " +
```

Comment:
endtime is by default null, how will this query behave for :windowStart <= endTime and endTime <= :windowEnd